seed/views/portfoliomanager.py

```diff
@@ -96,8 +104,11 @@ def report(self, request):
             properties = content_object['report']['informationAndMetrics']['row']
```

Comment:
It was great that the child data request XML objects came back in the same form once we get to this point, so there wasn't any major import code changes to make.